<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="User-PC" name="Test" tests="4" failures="1" timestamp="21 Jul 2020 11:28:22 GMT" time="56.973" errors="0">
  <testcase name="openHomePageNDTV" time="7.581" classname="Selenium.maven.Selenium.maven.demo.WeatherTest"/>
  <testcase name="verifyWeatherSection" time="2.038" classname="Selenium.maven.Selenium.maven.demo.WeatherTest"/>
  <testcase name="verifyWeatherSectionAPI" time="7.261" classname="Selenium.maven.Selenium.maven.demo.WeatherTest"/>
  <testcase name="verifyBothUIandAPISection" time="0.174" classname="Selenium.maven.Selenium.maven.demo.WeatherTest">
    <failure type="java.lang.AssertionError" message="expected [32] but found [Temp in Degrees: 35]">
      <![CDATA[java.lang.AssertionError: expected [32] but found [Temp in Degrees: 35]
	at Selenium.maven.Screens.WeatherPage.verifyUIAndApiDatsa(WeatherPage.java:98)
	at Selenium.maven.Selenium.maven.demo.WeatherTest.verifyBothUIandAPISection(WeatherTest.java:89)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- verifyBothUIandAPISection -->
</testsuite> <!-- Test -->
